package.json

```diff
@@ -21,26 +21,27 @@
     "url": "https://github.com/Sage/carbon/issues"
   },
   "homepage": "https://github.com/Sage/carbon#readme",
+  "peerDependencies": {
+    "flux": "^3.0.0",
```

Comment:
OK - I just wanted to ensure that applications are encouraged to upgrade. I don't expect them to be setting `^3.0.0` but instead to be locking their versions so they don't get unexpected upgrades. Since `3.1.1` had no meaningful updates it didn't seem like a bad idea to set that as the minimum.